h2/src/main/org/h2/expression/aggregate/AbstractAggregate.java

```diff
@@ -25,8 +25,14 @@
  */
 public abstract class AbstractAggregate extends DataAnalysisOperation {
 
+    /**
+     * is this a DISTINCT aggregate
+     */
     protected final boolean distinct;
 
+    /**
+     * FILTER condition for aggregate/window
```

Comment:
H2 supports `FILTER` clause only in aggregate functions (including their window versions, unlike some other databases). H2 does not support it in window functions.

SQL Standard also does not have such clause for window functions.